src/lib/mappers/charge-element.js

```diff
@@ -26,6 +26,7 @@ const dbToModelMapper = createMapper()
     'chargeVersionId'
   )
   .map().to('abstractionPeriod', abstractionPeriodMapper.dbToModel)
+  .map('factorsOverridden').to('isFactorsOverridden', factorsOverridden => factorsOverridden)
```

Comment:
I think the second argument to `.to()` can be omitted if you don't need to map the value

Reply:
Ah yes! I did have something going on in the function, but took it out. That looks quite silly 😂